### PR TITLE
Created CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by rafaelquint**
Added contributors of bash-scripts repo

Signed-off-by: Rafael Quintero <rquinter@redhat.com>

### CODE REVIEW

**Comment by timflannagan** [comment]
LGTM

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Rafael Quintero
```
Comment (adk3798) [inline CONTRIBUTORS.md:1]: Comrade Rafael Quintero

**Review by adk3798 — CHANGES_REQUESTED** [review]
Please Change

**Comment by nathan-weinberg** [comment]
So called "contributor" has not made any actual contributions

**Pull request closed without merging**